django-transparent/configs/fields.py

```diff
@@ -0,0 +1,51 @@
+from binascii import hexlify
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+import codecs
+
+class TextBinaryField(models.TextField):
```

Comment:
what about to add "Pg"\"Postgresql" prefix to field names? because now it's only pg compatible fields with prefixes '\\x'.
For mysql it should use literals like `X'<hex_value>'`

Reply:
Sure, good idea, thanks.